.travis.yml

```diff
@@ -1,29 +0,0 @@
----
-language: go
-go:
-  - 1.15.x
-services:
-  - docker
-install:
-  - make travis
-  - |
-    # Install ko
-    curl -L https://github.com/google/ko/releases/download/v0.8.0/ko_0.8.0_Linux_x86_64.tar.gz | tar xzf - ko
-    chmod +x ./ko
-    sudo mv ko /usr/local/bin
-
-script:
-  - set -e
-  - make
-  - make test-e2e TEST_IMAGE_REPO="$(./hack/install-registry.sh show):5000/shipwright-io/build-e2e"
-  - set +e
-deploy:
-  - provider: script
-    script: make release TAG=latest
-    on:
-      branch: master
-  - provider: script
-    script: make release TAG=$TRAVIS_TAG
```

Comment:
AFAIk this one is the one that allow us to build the https://quay.io/repository/shipwright/shipwright-operator?tab=tags , I´m not 100% sure if by removing this file we lose this TAG image generation.

Reply:
I see. The existence of this section is new to me, but just found this being ran for the v0.3.0 tag here: https://travis-ci.org/github/shipwright-io/build/builds/758198141. I'll bring back the file and only remove the `script` section as well as `make travis`.

Reply:
Done.

Reply:
nice, danke